src/containers/BellScheduleContainer.js

```diff
@@ -158,13 +160,15 @@ class BellScheduleContainer extends React.PureComponent<Props, State> {
           .hour(endHour)
           .minute(endMin);
         const current = now.diff(start) >= 0 && now.diff(end) < 0;
+        const percentThrough = now.diff(start) / end.diff(start);
 
         periods.push({
           period: scheduleData[periodTime],
           time: `${to12Hour(startHour)}:${startMin} - ${to12Hour(
             endHour
           )}:${endMin}`,
-          current: current
+          current: current,
+          percentThrough: percentThrough
```

Comment:
I'm not in love with the nomenclature of `percentThrough` have you considered `progress`? Perhaps you could get away with declaring it inline, too, as the additional constant declartation doesn't appear to be used anywhere else.